src/routes/game/components/GameMenu.vue

```diff
@@ -31,8 +31,11 @@
           <v-list-item data-cy="stalemate-initiate" @click="shownDialog = 'stalemate'">
             {{ t('game.menus.gameMenu.stalemate') }}
           </v-list-item>
-          <TheLanguageSelector />
         </template>
+        <TheLanguageSelector />
```

Comment:
I moved this out of the template for the v-else, this way you can still change the language if you are spectating